RxCocoa/Foundation/NSObject+Rx+RawRepresentable.swift

```diff
@@ -27,6 +27,40 @@ extension Reactive where Base: NSObject {
         return observe(E.RawValue.KVOType.self, keyPath, options: options, retainSelf: retainSelf)
             .map(E.init)
     }
+
+    /**
+     Specialization of generic `observe` method.
+
+     This specialization first observes `KVORepresentable` value and then converts it to `RawRepresentable` value.
+
+     It is useful for observing bridged ObjC enum values.
+
+     For more information take a look at `observe` method.
+     */
+    @available(swift 4.0)
+    public func observe<E: RawRepresentable>(_ keyPath: KeyPath<Base, E>, options: KeyValueObservingOptions = [.new, .initial], retainSelf: Bool = true) -> Observable<E?> where E.RawValue: KVORepresentable {
+        guard let keyPathString = keyPath._kvcKeyPathString else {
```

Comment:
Me too. But it's the only way to get the string for the new key path for now.

I think the long term solution would be to rewrite the entire thing to use the new [`observe(:)` API](https://github.com/apple/swift/blob/c5ff1f2cac8da6a14330f4b033b94c7c926d2126/stdlib/public/SDK/Foundation/NSObject.swift#L185). But I think it would be safer to go with wrappers for now.

Do you have any suggestion how to check or test?